### PR TITLE
Modified 404 & index and linked to new _layouts/default.html

### LINKED ISSUE #9: Design 404 page
## Problem statement
When the users are reaching an undefined URL on the new members site they face the default 404.html which is not really helping them to direct to the right place

## Design requirement
- Get the matching designs for the 404 page or use the existing one from a previous Harrow Woodcraft site

## Code requirement
- Implement the designs and test functionality

### PR DESCRIPTION
**Pull request by Linducya**
Created _layouts/default.html
Modified content of 404 page & linked to use default.html
Modified index.html to use default.html
Added link to logo in header.html

fixes #9

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <header>
   <div class="header__branding">
-    <img class="header__logo" src="img/harrow_woodcraft_logo.png" alt="The Harrow Woodcraft logo">
-    <h1 class="header__headline">HARROW WOODCRAFT FOLK</h1>
+   <a href="http://www.harrowwoodcraftfolk.btck.co.uk"><img class="header__logo" src="img/harrow_woodcraft_logo.png" alt="The Harrow Woodcraft logo"></a>
```
Comment (tansaku) [inline _includes/header.html:3]: great linkage here - I wonder if we should be linking to our home page rather than the other site?

**Review by tansaku — APPROVED** [review]

**Comment by tansaku** [comment]
this is wonderful @Linducya thanks so much it looks great - I was just going to add a screen shot, but I think I may have accidentally broken your site resolving a commit - ideally pull requests come from branches other than master - maybe I can show you how that all works on Friday ...

**Comment by tansaku** [comment]
thanks for fixing that @Linducya - screen shot of your great work:

File: _includes/header.html (hunk 1 above)
Comment (Linducya) [reply _includes/header.html:3]: As the same header.html is also already used by the home page, we thought to link the logo to the main harrow woodcraft site.

**Comment by Linducya** [comment]
Added Google Analytics Tracking ID UA-128490401-1 for test site linda.harrow-woodcraft-new-members.org.uk

This will need to change to UA-91621093-4 for harrow-woodcraft-new-members.org.uk

**Pull request closed without merging**